### PR TITLE
Attendance modify

### PR DESCRIPTION
**Pull request by aminmahrami**
Moved save request to seperate file and code cleaning.

### CODE REVIEW

File: src/components/Attendance/StudentWithWeeks.js [hunk 1]
```diff
@@ -31,12 +31,13 @@ export default class StudentWithWeeks extends React.Component{
     getIsHomeworkChecked = (id, student) => {
         if (this.props.allHistory[student][id].homework === 1) {
             return true;
-        } else {return false}
+        } else {return false;}
     };
 
     getIsAttendanceChecked = (id, student) => {
         if (this.props.allHistory[student][id].attendance === 1) {
```
Comment (M3kH) [inline src/components/Attendance/StudentWithWeeks.js]: ```return ( this.props.allHistory[student][id].attendance ===  1 )``` is enough.

File: src/components/Attendance/StudentWithWeeks.js [hunk 2]
```diff
@@ -31,12 +31,13 @@ export default class StudentWithWeeks extends React.Component{
     getIsHomeworkChecked = (id, student) => {
         if (this.props.allHistory[student][id].homework === 1) {
             return true;
-        } else {return false}
+        } else {return false;}
```
Comment (M3kH) [inline src/components/Attendance/StudentWithWeeks.js]: `return (this.props.allHistory[student][id].homework === 1)` is enough.
 Comparison operator already returns a boolean.

Comment (aminmahrami) [reply src/components/Attendance/StudentWithWeeks.js]: Thanks for Review and  Response @M3kH .

**Comment by aminmahrami** [comment]
Asked changes deployed.

**Pull request merged**